src/callbacks.jl

```diff
@@ -166,10 +166,17 @@ macro addLazyConstraint(cbdata, x)
     if length(x.args) == 3 # simple comparison
         lhs = :($(x.args[1]) - $(x.args[3])) # move everything to the lhs
         newaff, parsecode = parseExpr(lhs, :aff, [1.0])
+        lb, ub = 0.0, 0.0
+        if x.args[2] == :(<=) || x.args[2] == :≤
```

Comment:
Why do we need this new logic here? What about a ``construct_constraint`` method which takes a sense symbol?